ComponentKit/Core/Scope/CKComponentScopeFrame.mm

```diff
@@ -88,4 +88,29 @@ - (instancetype)initWithHandle:(CKComponentScopeHandle *)handle
   return self;
 }
 
+- (CKComponentScopeFrame *)scopeFrameForClass:(Class)componentClass identifier:(id)identifier;
+{
+  auto it = _children.find({componentClass, identifier});
```

Comment:
Updating the diff now; slightly concerned about complexity but it seems to work for my use case at the moment.